app/views/invoices/_form.html.slim

```diff
@@ -7,14 +7,14 @@
       placeholder: 'Customer',
       value: current_organization.find_customer_name_by_id(f.object.customer_id) } }
     = f.input :starts_at, as: :string, placeholder: 'Start date',
-      input_html: { class: 'datepicker' }
+      input_html: { class: 'datepicker', value: f.object.starts_at.try(:strftime, '%d/%m/%Y') }
```

Comment:
@yunixon use localize function for format date:
https://groups.google.com/forum/#!topic/plataformatec-simpleform/XcU4qx63TnM